packages/building-blocks/src/colors/blue.js

```diff
@@ -0,0 +1,16 @@
+const blue = {
+  50: "#E5ECF3",
+  100: "#CCDAE7",
+  200: "#B2C7DB",
+  300: "#99B5CF",
+  400: "#7FA2C2",
+  500: "#6690B7",
+  600: "#4D7EAB",
+  // 700 and above text is WCAG AA accessible on white backgrounds.
```

Comment:
I wonder if there's a way to integrate this comment into the storybook somehow. The stories themselves should provide all the documentation needed for someone to use these colors, including code samples!

Reply:
This is something that can come later, I'd be happy to get this merged in for now!